provider/core/Cargo.toml

```diff
@@ -36,6 +36,11 @@ std = ["icu_locid/std"]
 provider_serde = ["serde", "erased-serde"]
 macros = ["icu_provider_macros"]
 
+# Features for specific serde formats
+provider_json = ["serde", "serde_json"]
+provider_bincode1 = ["serde", "bincode", "std"]
+provider_postcard07 = ["serde", "postcard"]
```

Comment:
thought: cargo-all-features quickly explodes when we add tons of features, worth checking how long it takes to run on this crate and if we can reduce some of the combinations run (e.g. making `provider_*` all depend on `provider_serde`?)

Reply:
wfm!